pyproject.toml

```diff
@@ -17,7 +17,6 @@ pymysql = "^1.1.0"
 psycopg2-binary = "*"
 boto3 = "^1.28.62"
 python-dotenv = "^1.0.0"
-pyspark = "^3.5.0"
```

Comment:
Isn't it necessary anymore?

Reply:
you told me its in the main branch already?